### ISSUE TITLE
export images with red circle

### ISSUE THREAD

**Issue by SimpleDIY** [wrbl606/what-to-click#17]
It would be awesome if you could export the images with the red circle. Thank you for this great tool!

**Comment by pombredanne** [comment]
Agreed, right now this is an SVG overlay which is slick but it makes it harder to reuse images.
